Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+setuptools = "==69.5.1"
+wheel = "==0.43.0"
+numpy = "==1.26.4"
+astropy = "==6.0.1"
```

Comment:
Do you really need these specific version, or would the minor version be fine for compat? Since you've included the lock file anyway, it might be better to relax these requirements
```suggestion
setuptools = ">=69.5"
wheel = ">=0.43"
numpy = ">=1.26"
astropy = ">=6.0"
```
This will then use the same major version, with a lower bound on the minor version. Your package will play nicer with other people's work too, so they won't get into all sorts of headaches trying to install your work.

Reply:
Good point, I will relax these and also the dev dependancies.